### PR TITLE
Wrong class used in turbo streams

### PR DESCRIPTION
**Pull request by htmlshaman1**
TurboBundle::STREAM_FORMAT doesn't exist, that static variable was moved to TurboStreamResponse::STREAM_FORMAT.
Also the stream didn't work for me unless I changed:
$this->render('task/success.stream.html.twig', ['task' => $task]); to $this->render('task/success.stream.html.twig', ['task' => $task], new TurboStreamResponse());

| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no 
| Tickets       | Fix #... 
| License       | MIT

### CODE REVIEW

**Comment by weaverryan** [comment]
Hi!

Sorry about this. The documentation IS correct but not for the currently-released version. Let me be more clear:

* v2.0  (current version) uses `TurboStreamResponse` (as you've seen)
* v2.1 (unreleased) uses what you see in the documentation

What we need in the docs (and should have added when this change was originally made) was a `versionadded:: 2.1` tag that explained the change and how it worked before. For an example, see https://github.com/symfony/ux/blob/2.x/src/LiveComponent/src/Resources/doc/index.rst#L59

Would you be able to revert your changes by add a nice `versionadded` note explaining things?

Thanks!

**Review by htmlshaman1 — COMMENTED** [review]
Added a note about the class change from version 2.0 to 2.1

**Comment by htmlshaman1** [comment]
I added a brief message about the previous version using a different class, and returning a streaming response. If you can't use it, that's alright. I never knew contributing to the docs was so easy and smooth.

File: src/Turbo/Resources/doc/index.rst [hunk 1]
```diff
@@ -326,18 +326,23 @@ Let's discover how to use Turbo Streams to enhance your `Symfony forms`_::
                 // ... perform some action, such as saving the task to the database
 
                 // 🔥 The magic happens here! 🔥
-                if (TurboStreamResponse::STREAM_FORMAT === $request->getPreferredFormat()) {
+                if (TurboBundle::STREAM_FORMAT === $request->getPreferredFormat()) {
                     // If the request comes from Turbo, set the content type as text/vnd.turbo-stream.html and only send the HTML to update
-                    $request->setFormat(TurboStreamResponse::STREAM_FORMAT);
-                    return $this->render('task/success.stream.html.twig', ['task' => $task], new TurboStreamResponse());
+                    $request->setFormat(TurboBundle::STREAM_FORMAT);
+                    return $this->render('task/success.stream.html.twig', ['task' => $task]);
```
Comment (weaverryan) [inline src/Turbo/Resources/doc/index.rst]: Could you also revert these changes? We will only show the NEW way

File: src/Turbo/Resources/doc/index.rst [hunk 2]
```diff
@@ -326,18 +326,23 @@ Let's discover how to use Turbo Streams to enhance your `Symfony forms`_::
...
 
                 // If the client doesn't support JavaScript, or isn't using Turbo, the form still works as usual.
                 // Symfony UX Turbo is all about progressively enhancing your apps!
                 return $this->redirectToRoute('task_success', [], Response::HTTP_SEE_OTHER);
             }
 
-            // Symfony 5.3+
+  .. versionadded:: 2.1
+  
+  In versions prior to 2.1, TurboStreamResponse::STREAM_FORMAT was used instead of TurboBundle::STREAM_FORMAT. Also, one had to return a new TurboStreamResponse() object as the third argument for this->render().
```
Comment (weaverryan) [inline src/Turbo/Resources/doc/index.rst]: Let's move this above the code block, so people definitely see it.

Also, add manual line breaks around the 72nd character.

And put "ticks" (these characters: \`) around the technical words like `TurboStreamResponse::STREAM_FORMAT`.

**Review by weaverryan — COMMENTED** [review]
A few tweaks - thanks for making the changes :)

**Review by htmlshaman1 — COMMENTED** [review]
I think I have it.

**Review by htmlshaman1 — COMMENTED** [review]
Moved helpful message above code

**Comment by weaverryan** [comment]
Thanks @htmlshaman1!
